src/ManagedServiceIdentity/test/New-AzFederatedIdentityCredentials.Recording.json

```diff
@@ -0,0 +1,38 @@
+{
+  "New-AzFederatedIdentityCredentials+[NoContext]+CreateExpanded+$PUT+https://management.azure.com/subscriptions/0336439f-0e9d-44ec-975e-62accb9b3901/resourceGroups/fic-rg-yr1q7dts/providers/Microsoft.ManagedIdentity/userAssignedIdentities/fic-identity-6scr1l/federatedIdentityCredentials/fic-5ejdgy?api-version=2022-01-31-preview+1": {
+    "Request": {
+      "Method": "PUT",
+      "RequestUri": "https://management.azure.com/subscriptions/0336439f-0e9d-44ec-975e-62accb9b3901/resourceGroups/fic-rg-yr1q7dts/providers/Microsoft.ManagedIdentity/userAssignedIdentities/fic-identity-6scr1l/federatedIdentityCredentials/fic-5ejdgy?api-version=2022-01-31-preview",
+      "Content": "{\n  \"properties\": {\n    \"issuer\": \"https://kubernetes-oauth.azure.73dc98\",\n    \"subject\": \"system:serviceaccount:ns:svcaccount-9stnab\",\n    \"audiences\": [ \"api://AzureADTokenExchange\" ]\n  }\n}",
+      "isContentBase64": false,
+      "Headers": {
```

Comment:
nit: wondering why headers here are empty in comparison to get

Reply:
this is a mystery. but playback works